VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesActivity.java

```diff
@@ -26,11 +25,4 @@ public Fragment getFirstFragment() {
 
         return courseDiscussionResponsesFragment;
     }
-
-    @Override
-    protected void onStart() {
-        super.onStart();
-
-        setTitle(R.string.course_discussion_responses_title);
```

Comment:
Why not set the title from the Activity itself, as that is what is it ultimately is going to be set on?